doc/installation.rst

```diff
@@ -1,129 +1,217 @@
+============
 Installation
 ============
 
-There are several ways to get the IoTempower configuration management software up and running. **A native (classic) Linux installation is preferred and offers the best experience, maintainability, and full device access.** For Windows, we now recommend WSL2 with USB device support via `usbipd-win` (installed with winget). **Docker and Raspberry Pi installations are deprecated** due to technical and licensing issues.
+There are several ways to get the IoTempower configuration management software
+up and running. **A native (classic) Linux installation is always preferred**
+and offers the best experience, maintainability, and full device access.
+
+.. note::
+   **Transition Notice:** For Windows users, we now offer WSL2 with USB device
+   support via ``usbipd-win`` (installed using ``winget``) as a new recommended 
+   option. This provides a much better experience than Docker on Windows.
+   Docker and Raspberry Pi installations are being deprecated—see notes below.
+
 
-Linux, MacOS, Termux and WSL2 (Preferred)
-------------------------------------------
+Linux, MacOS, and Termux (Preferred)
```

Comment:
the preferred can be removed now it was a relic from WSL vs WSL 2.

However I would prefer if the "normal" Linux installation and WSL 2 installation were still mixed with hinting at the differences regarding USB flashing either via usbipd or using the mago and flashing via the network via the rfc method.